### PR TITLE
ReqIF import and re-import

### PR DESCRIPTION
**Pull request by franzTobiasDLR**
Implemented ReqIF requirements import and reimport.

### CODE REVIEW

File: de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/wizard/ReqIfImportWizard.java [hunk 1]
```diff
@@ -119,16 +121,40 @@ protected IStatus run(IProgressMonitor monitor) {
...
+		
+		CompoundCommand importCommand = new CompoundCommand();
+		importCommand.append(importer.persistSpecificationMapping(editingDomain, specMapping, reqIfContent, configurationContainer));
+		importCommand.append(importer.persistRequirementTypeContainer(editingDomain, typeContainer));
+		editingDomain.getCommandStack().execute(importCommand);
 
 	}
 	
+	/**
+	 * Do a re-import in a import configuration already exists. Existing requirement values are overwritten.
```
Comment (Anna-Lena-DLR) [inline de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/wizard/ReqIfImportWizard.java:162]: Do a re-import **if an** import configuration already exists.

File: de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/wizard/ReqIfMappingPage.java [hunk 2]
```diff
@@ -44,16 +48,21 @@
 	private static final String COLUMN_SPECIFICATION_LABEL = "Specification";
 	private static final String COLUMN_CONTAINER_LABEL = "Container";
 	
-	private static final int COLUMN_NAME_WIDTH = 300;
-	private static final int COLUMN_TYPE_WIDTH = 300;
+	private static final int COLUMN_NAME_WIDTH = 350;
+	private static final int COLUMN_TYPE_WIDTH = 350;
+	private static final int COMBO_TYPE_WIDTH = 500;
 
+	private static final String TYPE_CONTAINER_LABEL = "Requierement Type Container:";
```
Comment (Anna-Lena-DLR) [inline de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/wizard/ReqIfMappingPage.java:57]: "**Requirement** Type Container:";

**Review by Anna-Lena-DLR — APPROVED** [review]
Looks good :)

File: de.dlr.sc.virsat.target/tmodel-src/virsat.tmodel [hunk 3]
```diff
@@ -57,5 +57,6 @@ TargetLibrary virsat {
 		Unit org.eclipse.rmf.reqif10.feature.feature.group version 0.13.0.201509161042;
 		Unit org.eclipse.rmf.reqif10.pror.feature.feature.group version 0.13.0.201509161042;
 		Unit org.eclipse.rmf.reqif10.thirdparty.feature.feature.group version 0.13.0.201509161042;
+		Unit org.eclipse.rmf.reqif10.sdk.feature.feature.group version 0.13.0.201509161042;
 	}
 }
```
Comment (GeorgeNasrDLR) [inline de.dlr.sc.virsat.target/tmodel-src/virsat.tmodel:62]: Here it is complaining about the newline thing

**Review by GeorgeNasrDLR — COMMENTED** [review]
Great job (y)

**Review by GeorgeNasrDLR — APPROVED** [review]

**Pull request merged**